www/privacy/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page 
+title: Privacy Policy 
+---
+
+# Privacy Policy
```

Comment:
This is just the privacy policy for hte CLI. I think we should have that as a heading.